Program/internalexam.md

```diff
@@ -0,0 +1,83 @@
+### 9. Create text file on python by taking input from user. Read Python.txt file and Print it in Reverse. 
+
+
+```python
+f=open("D:\\23BCA299\\data\\python.txt","w")
+```
+
+
+```python
+line=[]
+```
+
+
+```python
+while True:
+    l=input()
+    if l:
+        line.append(l+"\n")
+    else:
+        break
+text="\n".join(line)
+```
+
+     python is a programming language
+     python is an interprited language
+     python is very easy language
+     python is a object-oriented language
+     
+    
+
+
+```python
+f.write(text)
+
+```
+
+
+
+
+    136
+
+
+
+
+```python
+f.close()
+```
+
+
+```python
+f=open("D:\\23BCA299\\data\\python.txt","r")
```

Comment:
we can also open file using "with statment". i request you to please make it and upload again

Reply:
ok i uplode file agin as par suggestion